modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -427,6 +429,32 @@ void ONNXImporter::addLayer(LayerParams& layerParams,
     }
 }
 
+void ONNXImporter::expandMid(const LayerParams& layerParams, opencv_onnx::NodeProto& node_proto,
+                             const std::string& input, size_t n, int axis)
+{
+    String base_name = layerParams.name + "/copy_";
+    std::vector<std::string> input_names;
```

Comment:
Use `std::vector<String>`.

You know the number of elements - add `input_names.reserve(n)` call

Reply:
done